AppController/scripts/controller

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+### BEGIN INIT INFO
+# Provides:          controller
+# Required-Start:    $local_fs
+# Required-Stop:     $local_fs
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# X-Interactive:     false
+# Short-Description: Init script for AppScale controller.
+# Description:       Start/stop script for AppScale controller.
+### END INIT INFO
+
+DESC="AppScale Controller"
+DIR=/root/appscale/AppController
+DAEMON=$DIR/djinnServer.rb
+DAEMON_NAME=controller
```

Comment:
This may be just a personal preference, but "controller" seems too generic for a daemon name.